src/types.ts

```diff
@@ -1147,14 +1148,24 @@ export interface UpdateCredentialsPayload {
     currentNumberOfCredentials: bigint;
 }
 
+export interface ConfigureDelegationPayload {
+    /* stake to delegate. if set to 0, this removes the account as a delegator */
+    stake?: GtuAmount;
+    /* should earnings from delegation be added to staked amount  */
+    restakeEarnings?: boolean;
+    /* determines which baker (or the L-pool), the account should delegate to  */
```

Comment:
```suggestion
    /* determines which baker (if not passive delegation), the account should delegate to  */
```

Reply:
Of course! changed the entire sentence though, to:
"determines if the account should use passive delegation, or which specific baker to delegate to"